src/ExtensionsSample/Program.cs

```diff
@@ -30,7 +30,6 @@ public static void Main(string[] args)
             config.UseSample();
             config.UseMobileApps();
             config.UseTwilioSms();
-            config.UseCore();
```

Comment:
There are Timer/Files samples that rely on this. I think you should leave this. I want to keep the samples running.

Reply:
The only thing core was doing was the registration of the execution context binding provider, right? 
https://github.com/Azure/azure-webjobs-sdk-extensions/blob/7e00bdbcefc1cfcae682a2a089af0dcef1bd992b/src/WebJobs.Extensions/Extensions/Core/Config/CoreJobHostConfigurationExtensions.cs#L36-L45